subprojects/docs/src/samples/user-input/pass-credentials-to-external-tool-via-stdin/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+val login = tasks.register<Exec>("login") {
+    val username: String by project
+    val password: String by project
+    standardInput = java.io.ByteArrayInputStream("$username\n$password".toByteArray())
+    commandLine = listOf("sh", "login.sh")
```

Comment:
I *think* just setting `executable = "login.sh"` might work too?

Reply:
Nope, that's what I tried initially - does not compute :(

Now I'm thinking - if we are ok with the concept of such sample overall - maybe either convert the bash script to a Java application with same behavior (perhaps making it less scary and more familiar to Windows users), or build one more sample using JavaExec task for the same?

Reply:
Actually, `executable = "./login.sh"` works.